src/sentry/static/sentry/app/components/actions/actionLink.tsx

```diff
@@ -47,16 +45,22 @@ export default class ActionLink extends React.Component {
       );
     } else {
       return (
-        <a
+        <ActionLinkAnchor
           data-test-id={testId}
           aria-label={title}
           className={classNames(className, {disabled})}
           onClick={disabled ? undefined : onAction}
           disabled={disabled}
         >
           {children}
-        </a>
+        </ActionLinkAnchor>
       );
     }
   }
 }
+
+const ActionLinkAnchor = styled('a')<{
```

Comment:
Kind of a style nit, but I've just been writing these as single lines

```jsx
const ActionLinkAnchor = styled('a')<{disabled?: boolean;}>`
...
`
```

Reply:
👍